core/operation/src/test/java/uk/gov/gchq/gaffer/operation/OperationChainTest.java

```diff
@@ -66,11 +71,11 @@ public void shouldSerialiseAndDeserialiseOperationChain() throws SerialisationEx
         assertNotNull(deserialisedOp);
         assertEquals(2, deserialisedOp.getOperations().size());
         assertEquals(OperationImpl.class, deserialisedOp.getOperations()
-                                                        .get(0)
```

Comment:
Do we need to agree a standard for formatting? Or at least make sure that everything is using the agreed specification? I've been seeing quite a few places in PRs recently where there are formatting differences.